src/arguments/CoreEmoji.ts

```diff
@@ -10,12 +9,12 @@ export class CoreArgument extends Argument<EmojiObject> {
 
 	public run(parameter: string): ArgumentResult<EmojiObject> {
 		const resolved = resolveEmoji(parameter);
-		return resolved.success
-			? this.ok(resolved.value)
-			: this.error({
-					parameter,
-					identifier: Identifiers.ArgumentEmojiError,
-					message: 'The argument did not resolve to an emoji.'
-			  });
+		return resolved.mapErrInto((identifier) =>
+			this.error({
+				parameter,
+				identifier,
+				message: 'The argument did not resolve to an emoji.'
+			})
```

Comment:
I just realised, `context` is not passed in this argument, while it is everywhere else. 😅